src/posh/core.cljc

```diff
@@ -31,34 +31,34 @@
    :schemas {}
    :txs {}
    :dbs {}
-   :filters {}
+   :filters {}})
    ;; {db-id {:filter pred :as-of t :with tx-data :since t}}
-   })
+
 
 (defn add-db
   ([posh-tree db-id conn schema] (add-db posh-tree db-id conn schema nil))
   ([{:keys [dcfg conns schemas dbs cache graph] :as posh-tree}
     db-id conn schema base-filters]
-     (let [storage-key [:db db-id]]
```

Comment:
Sorry about all the whitespace noise. Parinfer is an unforgiving (though generous) mistress.